cds_books/ldap/admin.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# CDS Books is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Admin panel for ldap synchronizations."""
+
+import flask
+from flask import Blueprint, flash, redirect, request
+from flask_admin.base import expose
+from flask_admin.contrib.sqla import ModelView
+from flask_admin.helpers import get_redirect_target
+from flask_babelex import gettext as _
+from invenio_db import db
+
+from cds_books.ldap.models import LdapSynchronizationLog
+from cds_books.ldap.tasks import synchronize_users_task
+
+
+class LdapSynchronizationLogModelView(ModelView):
+    """Invenio admin view for ldap users synchronization."""
+
+    can_create = True
+    can_edit = False
+    can_delete = False
+    can_view_details = True
+    form_excluded_columns = LdapSynchronizationLog.__table__.columns
+    column_default_sort = (LdapSynchronizationLog.start_time, True)
+
+    @expose('/new/', methods=('GET', 'POST'))
+    def create_view(self):
```

Comment:
@FlorianCassayre I think we might have a problem with permissions here. Have you checked that only authenticated admin users can post on this one?

Reply:
I recall trying to access this very page as an anonymous user, and the system would redirect me to the login form. I'm assuming it's handled internally by `flask-admin`, but I'll double check.

Reply:
I confirm that, as for the other pages, it:
- redirects to login page when not logged-in
- returns `403` when logged-in as a patron 
![image](https://user-images.githubusercontent.com/9027075/90229350-10c2ba80-de18-11ea-9859-7f12c4d6b652.png)